app/src/main/java/com/ciandt/dragonfly/example/infrastructure/extensions/Int+Extension.kt

```diff
@@ -0,0 +1,7 @@
+package com.ciandt.dragonfly.example.infrastructure.extensions
+
+import android.content.res.Resources
+
+fun Int.toPx(): Int = (this * Resources.getSystem().displayMetrics.density).toInt()
```

Comment:
I think px(), instead of toPx() is better. Just a suggestion... (same for toDp())